CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Release 136 (2022-08-23)
+------------------------
+Add nl.basjes/yauaa_context/jsonschema/1-0-4 (#1216)
+Extend copyright to 2022 (close #1218)
+Update links in Readme (close #1219)
+Add CONTRIBUTING.md (close #1220)
```

Comment:
nit: I don't think we usually have the word `close` in these lines

Reply:
Done.